dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -120,6 +120,134 @@
 			label = "GPT2";
 		};
 
+		tmr1: qtmr@401dc000 {
```

Comment:
I'd prefer to use labels like `qmtr1` here (as opposed to `tmr1), to follow the pattern used by the remainder of the nodes in the DTSI file.

Reply:
Changed.